src/main/java/org/folio/rest/service/report/RefundReportService.java

```diff
@@ -464,10 +464,16 @@ private static String singleOrDefaultMessage(List<String> values, String message
   }
 
   private static String getStaffInfo(String comments) {
+    if(comments == null){
```

Comment:
Just to preserve our existing code style, can we add spaces here? Like in `if (condition()) {`